tests/issues.targets

```diff
@@ -196,6 +196,9 @@
         <ExcludeList Include="$(XunitTestBinBase)\baseservices\varargs\varargsupport_r\varargsupport_r.cmd">
             <Issue>Varargs supported on this platform</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)\Interop\COM\NETClients\Primitives\NETClientPrimitives\NETClientPrimitives.cmd">
```

Comment:
@BruceForstall If this is the right place to disable then please remove the entry on line 439 - which was intended to disable this test when being run in helix - it consistently runs properly in the official coreclr CI.